.github/workflows/tarpaulin.yml

```diff
@@ -0,0 +1,66 @@
+name: tarpaulin
+
+on:
+  pull_request:
+    paths:
+      - Cargo.lock
+      - Cargo.toml
+      - src/*.rs
+      - src/**/*.rs
+      - src/**/**/*.rs
+
+permissions:
+  contents: read
+  pull-request: write
+
+jobs:
+  coverage:
+    name: coverage
+    permissions:
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+      - name: python
```

Comment:
For the PR comment, we use a Python CLI to format Tarpaulin's output.  Thus, we need to configure a Python environment.